package.json

```diff
@@ -55,19 +55,21 @@
     "@babel/plugin-proposal-object-rest-spread": "7.3.4",
     "@babel/plugin-transform-block-scoping": "7.3.4",
     "@babel/plugin-transform-modules-commonjs": "7.2.0",
-    "@babel/plugin-transform-flow-strip-types": "7.3.4",
-    "@babel/plugin-transform-react-jsx": "7.3.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
```

Comment:
We should use the react preset because it has few additional transforms such as adding `displayName` etc.

The typescript and flow presets are basically just the transforms, nothing else.